doc/release-notes-1.10.rst

```diff
@@ -20,22 +20,39 @@ The following sections provide detailed lists of changes by component.
 Kernel
 ******
 
-* details ...
+* Remove deprecated k_mem_pool_defrag code
+* add memory domain APIs Add the following application-facing memory domain
+  APIs:
+  * k_mem_domain_init() - to initialize a memory domain
```

Comment:
For brevity, you could just refer folks to the API documentation for these k_mem_domain functions:  http://docs.zephyrproject.org/api/kernel_api.html#memory-domain